2.0/flask_app/routes/rag.py

```
@@ -0,0 +1,24 @@
+from flask import Blueprint, request, jsonify
+import logging
+
+from config.log_config import config_logging
+from services.llm_service import RAGService
+
+rag_bp = Blueprint('rag', __name__, url_prefix='/api/rag')
+rag_service = RAGService()
+config_logging('web') ### TESTING. Maybe move to routes __init__???
+
+@rag_bp.route('/query', methods=['POST'])
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)